contracts/ArchetypeLogic.sol

```diff
@@ -162,6 +163,11 @@ library ArchetypeLogic {
     }
 
     uint256 cost = price * numTokens;
+    
+    if(invite.interval == 0){ //Apply the linear curve 
```

Comment:
@kneelsdev this condition seems meaningless, considering all invite.intervals default to 0.

Reply:
The interval is used for dutch invite functionality, we can clean this up to make the three scenarios more clear:
```
if interval >0 & delta > 0:
  dutch invite pricing
else if interval == 0 & delta > 0:
  linear curve pricing
else
  regular pricing
```